README.md

```
@@ -194,6 +194,71 @@ Analyze CoW Swap bundle:
 ANALYZE_BUNDLE=true ANALYZE_BUNDLE_TEMPLATE=sunburst pnpm run build
 ```
 
+## Developing against a local `cow-sdk` checkout
+
+Sometimes a feature needs `@cowprotocol/cow-sdk` changes that aren't published yet (e.g. the
+Solana trading support under active development). To develop against a sibling
+[`cow-sdk`](https://github.com/cowprotocol/cow-sdk) checkout instead of the published npm
+packages, clone it next to this repo:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)